tokio-trace/src/macros.rs

```diff
@@ -42,34 +54,64 @@
 /// # }
 /// ```
 ///
-/// Creating a span with custom target and log level:
+/// As a shorthand, local variables may be used as field values without an
+/// assignment, similar to [struct initializers]. For example:
 /// ```
 /// # #[macro_use]
 /// # extern crate tokio_trace;
 /// # use tokio_trace::Level;
 /// # fn main() {
-/// span!(
-///     Level::TRACE,
-///     target: "app_span",
-///     "my span",
-///     foo = 3,
-///     bar = "another string"
-/// );
+/// let user = "ferris";
+/// span!(Level::TRACE, "login", user);
```

Comment:
Same here -- it'd be useful to show both with and without the shorthand to make it crystal clear what the difference is.